.storybook/recipes/DataSummaryCard/DataSummaryCard.module.css

```diff
@@ -0,0 +1,106 @@
+@import '../../../src/design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # DATA SUMMARY CARD
+\*------------------------------------*/
+
+/**
+ * Data Summary Card centers and wraps header, data, and footnote.
+ */
+.data-summary-card {
+  width: 12.75rem;
+  height: 9.25rem;
+  position: relative;
+  overflow: hidden;
+
+  border: 0;
+  box-shadow: var(--eds-box-shadow-sm);
+
+  align-items: center;
+}
+.data-summary-card:hover {
+  border: var(--eds-border-width-sm) solid
+    var(--eds-theme-color-border-neutral-default);
+}
+
+/**
+ * The header wraps the title.
+ */
+.data-summary-card__header {
+  margin-top: var(--eds-size-2);
+  margin-bottom: var(--eds-size-half);
+}
+
+/**
+ * Prevents jitter of the card components when hovering due to border spawning.
+ * Transparent border can't be used due to off-track variant indicator needing full width.
+ */
+.data-summary-card:hover > .data-summary-card__header {
+  margin-top: calc(var(--eds-size-2) - var(--eds-border-width-sm));
+}
+
+/**
+ * The title communicates card organization.
+ */
+.data-summary-card__title {
+  @mixin eds-theme-typography-overline-sm;
+}
+
+/**
+ * The body wraps data and description text.
+ */
+.data-summary-card__body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+/**
+ * The data is the main content for the card.
+ */
+.data-summary-card__data {
+  @mixin eds-theme-typography-headline-lg;
+
+  display: flex;
+  align-items: center;
+  gap: var(--eds-size-half);
+
+  margin-bottom: var(--eds-size-half);
+}
+
+/**
+ * The data unit provides a unit of measurement for the data to make it relevant.
+ */
+.data-summary-card__data-unit {
+  @mixin eds-theme-typography-headline-sm;
+}
+
+/**
+ * The Description provides more elucidation for the data.
+ */
+.data-summary-card__description {
+  @mixin eds-theme-typography-caption-text-sm;
+  font-weight: var(--eds-font-weight-light);
+}
+
+/**
+ * Off track variant, bottom orange border indicates off track status.
+ */
+.data-summary-card__indicator--off-track {
+  width: 100%;
+  height: 8px;
+
+  position: absolute;
+  bottom: 0;
+  left: 0;
+
+  background-color: var(--eds-theme-color-icon-utility-warning);
+}
+
+/**
+ * Prevents jitter of the card components when hovering due to border spawning.
+ * Transparent border can't be used due to off-track variant indicator needing full width.
+ */
+.data-summary-card:hover > .data-summary-card__indicator--off-track {
+  height: 7px;
+}
```

Comment:
Please advise on a lighter way to prevent jitter if possible.
Tried doing `border-color: transparent` and just `border-bottom: 8px var(--eds-theme-color-orange)` but there's noticeable side borders 
![image](https://user-images.githubusercontent.com/86632227/187587249-e3526ee5-895b-4f23-a8f9-c7934ea470ab.png)

Reply:
I think this is fine. I know it feels kind of hacky, but it doesn't look that bad to me. Thanks for paying attention to these finer details ❤️